### DISCUSSION TITLE
How to remove second car from database?

### DISCUSSION THREAD

**Discussion by cleanev** [teslamate-org/teslamate#2613] [Q&A] [answered]
Hello all,

Currently using latest 1.26.1 version.

I added additional driver who also owns a Tesla Y to my account and after about 4-5 months found that my car has been updating both implementations of Teslamate. Is there a way to remove data for second car from another instance? 

In below snapshot even after removing second vehicle old data still shows (bottom is second vehicle, that is now removed)
![0916417F-8DE0-4B48-AD12-FD3DD1B48425](https://user-images.githubusercontent.com/45519661/164774216-fc9e1460-15b7-4ddf-9469-f4fa71aa050c.jpeg)

**Comment by cleanev** [comment]
Pinging @Dulanic & @DrMichael whom I have seen posting scripts on this forum. If either of you can shed some light on how to cleanup database for the car that is now removed from the account?

**Comment by Dulanic** [comment, marked as answer]
Follow this comment, it's better than my recommendation.   https://github.com/adriankumpf/teslamate/issues/706#issuecomment-705370576

**Reply by cleanev** [reply]
Thanks a lot. Link your provided perfectly and database size decreased as well.
